src/main/java/kz/toko/app/entity/UserEntity.java

```diff
@@ -0,0 +1,44 @@
+package kz.toko.app.entity;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.validation.constraints.NotEmpty;
+
+@Data
+@Entity
+@Table(name = "users")
+public class UserEntity extends AuditableEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    @SequenceGenerator(name = "sequenceGenerator")
```

Comment:
Either rename sequence, or use `Identity` as I did

Reply:
changed to Identity